h2o-r/h2o-package/pkgdown/_pkgdown.yml

```diff
@@ -287,7 +287,7 @@ reference:
       - h2o.scoreHistoryGAM
       - h2o.sd 
       - h2o.sdev
-      - h2o.dt
+      - h2o.decisionTree
```

Comment:
Please change this to `h2o.decision_tree()`.  We retired camelCase from our R package many years ago and now follow the Tidyverse style guide, which uses snake case for new functions.  Thanks!

Reply:
Ok, thanks for info. Resolved